templates/package.json.tpl

```diff
@@ -1,6 +1,7 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
+    "@outreach/semantic-release-circleci-orb": "^1.1.9",
```

Comment:
It's not required by everything though... so it's not great to throw it everywhere.

The better solution, I think, would be to parse the existing file and merge in our dependencies. We could do this in the go-template today, if you'd like to pair on it tomorrow.